adapters/kargo/params_test.go

```diff
@@ -0,0 +1,43 @@
+package kargo
+
+import (
+	"encoding/json"
+	"testing"
+
+	"github.com/prebid/prebid-server/openrtb_ext"
+)
+
+func TestValidParams(t *testing.T) {
+	validator, err := openrtb_ext.NewBidderParamsValidator("../../static/bidder-params")
+	if err != nil {
+		t.Fatalf("Failed to fetch the json schema. %v", err)
+	}
+
+	for _, p := range validParams {
+		if err := validator.Validate(openrtb_ext.BidderKargo, json.RawMessage(p)); err != nil {
+			t.Errorf("Schema rejected valid params: %s", p)
+		}
+	}
+}
+
+func TestInvalidParams(t *testing.T) {
+	validator, err := openrtb_ext.NewBidderParamsValidator("../../static/bidder-params")
+	if err != nil {
+		t.Fatalf("Failed to fetch the json schema. %v", err)
+	}
+
+	for _, p := range invalidParams {
+		if err := validator.Validate(openrtb_ext.BidderKargo, json.RawMessage(p)); err == nil {
+			t.Errorf("Schema allowed invalid params: %s", p)
+		}
+	}
+}
+
+var validParams = []string{
+	`{"placementId": ""}`,
+	`{"placementId": "_mx764d"}`,
+}
+
+var invalidParams = []string{
+	`{"placementId": 42}`,
```

Comment:
Can we include more test scenarios inside `adapters/kargo/params_test.go`'s `invalidParams` array? Maybe something like:
```
   var invalidParams = []string{
       `{"placementId": 42}`,
 +     `{"placementId": }`,
 +     `{}`,
   }
```